MIGRATION.md

```diff
@@ -8,6 +8,50 @@ npx @sentry/migr8@latest
 
 This will let you select which updates to run, and automatically update your code. Make sure to still review all code changes!
 
+## Deprecate `Hub`
+
+The `Hub` has been a very important part of the Sentry SDK API up until now.
+Hubs were the SDK's "unit of concurrency" to keep track of data across threads and to scope data to certain parts of your code.
+Because it is overly complicated and confusing to power users, it is going to be replaced by a set of new APIs: the "new Scope API".
+
+`Scope`s have existed before in the SDK but we are now expanding on them because we have found them powerful enough to fully cover the `Hub` API.
+
+If you are using the `Hub` right now, see the following table on how to migrate to the new API:
+
+| Old `Hub` API | New `Scope` API |
+| --- | --- |
+| `new Hub()` | `withScope()`, `withIsolationScope()` or `new Scope()` |
+| hub.isOlderThan() | REMOVED - Was used to compare `Hub` instances, which are gonna be removed. |
+| hub.bindClient() | A combination of `scope.setClient()` and `client.setupIntegrations()` |
+| hub.pushScope() | Best replaced with `withScope()` |
+| hub.popScope() | When used in combination with `hub.pushScope()` best replaced with `withScope()` |
+| hub.withScope() | `withScope()` |
+| getClient() | `scope.getClient` |
+| getScope() | REMOVED - Scopes are used directly now. |
+| getIsolationScope() | `Sentry.getIsolationScope()` |
+| getStack() | REMOVED - The stack used to hold scopes. Scopes are used directly now. |
+| getStackTop() | REMOVED - The stack used to hold scopes. Scopes are used directly now. |
+| captureException() | `scope.captureException()` |
+| captureMessage() | `scope.captureMessage()` |
+| captureEvent() | `scope.captureEvent()` |
```

Comment:
```suggestion
| captureException() | `Sentry.captureException()` |
| captureMessage() | `Sentry.captureMessage()` |
| captureEvent() | `Sentry.captureEvent()` |
```

IMHO users should avoid using these on the scope directly, just use the global methods.